### PR TITLE
added alternate solution to multiply using Exponentiation operator

### PR DESCRIPTION
**Pull request by trekitch**
Added a small alternative solution to one of the problems in the calculator project.

### CODE REVIEW

**Comment by xandora** [comment]
The first problem I see here is that this code *should* still run. Will it run with two functions with the same name?

**Comment by trekitch** [comment]
@xandora I was under the impression that the Jest solutions branch was just used to show the solutions. I got the idea for the alternate solution from the recursive factorial alternate solution. I do see that that one is name different but still not used. I have update my alternate function to reflect that.

File: calculator/calculator.js [hunk 1]
```diff
@@ -20,6 +20,11 @@ const power = function(a, b) {
   return Math.pow(a, b);
 };
 
+//alternate solution using Exponentiation opertator
```
Comment (xandora) [inline calculator/calculator.js:23]:
```suggestion
//alternate solution using the Exponentiation operator
```

**Review by xandora — CHANGES_REQUESTED** [review]
Just one small change required, but this looks good to me!

**Comment by trekitch** [comment]
Accepted changes

**Pull request merged**